monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -356,6 +361,11 @@ class ReportPageComponent extends AuthComponent {
         overviews.push(this.getIssueOverview(this.IssueDescriptorEnum[issues[i]]));
       }
     }
+
+    if (this.tunnelingIssueExists === true) {
+      overviews.push(this.getIssueOverview(this.IssueDescriptorEnum['tunnel']))
```

Comment:
Always push the tunneling issue overview. The overview will return `<></>` if no tunneling happened